app/main.py

```diff
@@ -1,8 +1,19 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = [Person(human["name"], human["age"]) for human in people]
+    for human in people:
+        if human.get("wife"):
+            Person.people[human["name"]].wife = Person.people[human["wife"]]
+        if human.get("husband"):
+            Person.people[human["name"]].husband \
+                = Person.people[human["husband"]]
```

Comment:
use `()` instead of `\`

Reply:
![str2](https://github.com/mate-academy/py-person-class/assets/136181910/4dd0011d-d0b9-40d4-9d0a-f1ff6fb44e70)
Pycharm raises error

Reply:
if you can just show how to do it right way